WordCounter/Histogram.cs

```diff
@@ -0,0 +1,31 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace WordCounterProject
+{
+    class Histogram
+    {
+        public Histogram() { }
+
+        public static void Start(Dictionary<string, int> dict)
```

Comment:
Public methods / functions should always have documentation.

Example:

/// <summary>
///
/// </summary>
/// <example>
/// <code>
/// IDictionary dict = new Dictionary<string, int>() {
///   { "the", 18 },
///   { "to", 20 },
/// };
/// Histogram.Start(dict);
/// </code>
/// </example>
/// <param name="dict">A dictionary where each key represents a word, and the value is the number of times the word was found in a text.</param>